.github/workflows/assemble.yml

```diff
@@ -30,8 +30,11 @@ jobs:
       - name: Setup docker (missing on MacOS)
         id: setup_docker
         if: runner.os == 'macos'
+        continue-on-error: true
         run: |
-          exit 0;
+          brew install docker colima coreutils
+          gtimeout 15m colima start
```

Comment:
To mitigate possible future issues with hanging step(s), capping it to 15m

Reply:
@reta Thanks for limiting the time just in case, looks reasonable to me.